CI fails: ImportError: cannot import name 'ResponseTextConfig' from 'openai.types.responses.response'

CI fails: https://github.com/huggingface/smolagents/actions/runs/17047167769/job/48326045318?pr=1691
> ImportError: cannot import name 'ResponseTextConfig' from 'openai.types.responses.response'

It seems `litellm` is broken after latest 1.100.0 `openai` release:
- https://github.com/BerriAI/litellm/issues/13711

Stack trace:
```python
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/smolagents/models.py:1127: in __init__
    super().__init__(
/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/smolagents/models.py:1072: in __init__
    self.client = client or self.create_client()
/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/smolagents/models.py:1137: in create_client
    import litellm
/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/litellm/__init__.py:20: in <module>
    from litellm.llms.custom_httpx.http_handler import AsyncHTTPHandler, HTTPHandler
/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/litellm/llms/custom_httpx/http_handler.py:16: in <module>
    from litellm.litellm_core_utils.logging_utils import track_llm_api_timing
/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/litellm/litellm_core_utils/logging_utils.py:7: in <module>
    from litellm.types.utils import (
/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/litellm/types/utils.py:42: in <module>
    from ..litellm_core_utils.core_helpers import map_finish_reason
/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/litellm/litellm_core_utils/core_helpers.py:8: in <module>
    from litellm.types.llms.openai import AllMessageValues
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

    from enum import Enum
    from os import PathLike
    from typing import IO, Any, Iterable, List, Literal, Mapping, Optional, Tuple, Union
    
    import httpx
    from openai._legacy_response import (
        HttpxBinaryResponseContent as _HttpxBinaryResponseContent,
    )
    from openai.lib.streaming._assistants import (
        AssistantEventHandler,
        AssistantStreamManager,
        AsyncAssistantEventHandler,
        AsyncAssistantStreamManager,
    )
    from openai.pagination import AsyncCursorPage, SyncCursorPage
    from openai.types import Batch, EmbeddingCreateParams, FileObject
    from openai.types.beta.assistant import Assistant
    from openai.types.beta.assistant_tool_param import AssistantToolParam
    from openai.types.beta.thread_create_params import (
        Message as OpenAICreateThreadParamsMessage,
    )
    from openai.types.beta.threads.message import Message as OpenAIMessage
    from openai.types.beta.threads.message_content import MessageContent
    from openai.types.beta.threads.run import Run
    from openai.types.chat import ChatCompletionChunk
    from openai.types.chat.chat_completion_audio_param import ChatCompletionAudioParam
    from openai.types.chat.chat_completion_content_part_input_audio_param import (
        ChatCompletionContentPartInputAudioParam,
    )
    from openai.types.chat.chat_completion_modality import ChatCompletionModality
    from openai.types.chat.chat_completion_prediction_content_param import (
        ChatCompletionPredictionContentParam,
    )
    from openai.types.embedding import Embedding as OpenAIEmbedding
    from openai.types.fine_tuning.fine_tuning_job import FineTuningJob
>   from openai.types.responses.response import (
        IncompleteDetails,
        Response,
        ResponseOutputItem,
        ResponseTextConfig,
        Tool,
        ToolChoice,
    )
E   ImportError: cannot import name 'ResponseTextConfig' from 'openai.types.responses.response' (/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/openai/types/responses/response.py)

/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/litellm/types/llms/openai.py:36: ImportError
```

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -46,6 +46,7 @@ gradio = [
 ]
 litellm = [
   "litellm>=1.60.2",
+  "openai<1.100.0",
 ]
 mcp = [
   "mcpadapt>=0.1.11",  # Support Image and Audio content

